ci/deploy.sh

```diff
@@ -48,8 +49,8 @@ This build is based on
 https://github.com/$TRAVIS_REPO_SLUG/commit/$TRAVIS_COMMIT.
 
 This commit was created by the following Travis CI build and job:
-https://travis-ci.org/$TRAVIS_REPO_SLUG/builds/$TRAVIS_BUILD_ID
-https://travis-ci.org/$TRAVIS_REPO_SLUG/jobs/$TRAVIS_JOB_ID
+https://travis-ci.com/$TRAVIS_REPO_SLUG/builds/$TRAVIS_BUILD_ID
```

Comment:
What do we want to do with manuscripts still on travis-ci.org?  Should we leave this?

Reply:
I will revert this change. Manuscripts still on travis-ci.org should still use the .org link. However, since it hasn't been a conflict, I've missed it in my previous upgrades.